<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>648</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1071</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="Line" name="line_2">
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QWidget" name="Yield_calculation" native="true">
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>210</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>到期收益率（买入）</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="yield_at_buy">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>210</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="buy_date_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>买入日期</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="buy_date_text_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="buy_date_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>赎回/回购</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="option">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="buy_date_4">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>80</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>应计利息</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>250</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>行权收益率</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="yield_if_exercised">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>250</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="buy_date_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>120</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>净价</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="clean">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="buy_date_6">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>160</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>全价</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_clear_yield">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>210</y>
         <width>121</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_yield">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>240</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>开始计算</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="accrued">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>80</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="full">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>160</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="Line" name="line_4">
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="Quote" native="true">
      <widget class="QPushButton" name="pushButton_quote">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>112</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>获取行情</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>331</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>银行间</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>上交所</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>深交所</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>代码</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTextBrowser" name="code_ib"/>
        </item>
        <item row="1" column="2">
         <widget class="QTextBrowser" name="code_sh"/>
        </item>
        <item row="1" column="3">
         <widget class="QTextBrowser" name="code_sz"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>净价</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QTextBrowser" name="clean_price_ib"/>
        </item>
        <item row="2" column="2">
         <widget class="QTextBrowser" name="clean_price_sh"/>
        </item>
        <item row="2" column="3">
         <widget class="QTextBrowser" name="clean_price_sz"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>全价</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QTextBrowser" name="full_price_ib"/>
        </item>
        <item row="3" column="2">
         <widget class="QTextBrowser" name="full_price_sh"/>
        </item>
        <item row="3" column="3">
         <widget class="QTextBrowser" name="full_price_sz"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>收益率</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QTextBrowser" name="yield_ib"/>
        </item>
        <item row="4" column="2">
         <widget class="QTextBrowser" name="yield_sh"/>
        </item>
        <item row="4" column="3">
         <widget class="QTextBrowser" name="yield_sz"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="HPY" native="true">
      <widget class="QDateEdit" name="sell_date_text">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>160</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>130</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>HPY（年化）</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>80</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>HPY</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="sell_clean_price_text">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>120</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="hpy_annualized_text">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>130</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="sell_clean_price">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>卖出净价</string>
       </property>
      </widget>
      <widget class="QLabel" name="buy_clean_price">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>买入净价</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="buy_clean_price_text">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="sell_date">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>160</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>卖出日期</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="buy_date_text">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="hpy_text">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>80</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="buy_date">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>买入日期</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>180</y>
         <width>171</width>
         <height>100</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_repo_hpy">
          <property name="text">
           <string>开始计算(买断式回购)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_hpy">
          <property name="text">
           <string>开始计算</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_clear">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line">
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Line" name="line_3">
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="Basic_Info" native="true">
      <widget class="QLabel" name="code">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>债券代码</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="code_text">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_basicinfo">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>112</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>获取信息</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>251</width>
         <height>201</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="1">
         <widget class="QTextBrowser" name="category"/>
        </item>
        <item row="4" column="1">
         <widget class="QTextBrowser" name="settlement"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>结算速度</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>交易平台</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTextBrowser" name="location"/>
        </item>
        <item row="1" column="1">
         <widget class="QTextBrowser" name="platform">
          <property name="maximumSize">
           <size>
            <width>188</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>上市地点</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>报价方式</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>债券类别</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QTextBrowser" name="quote_convention"/>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>110</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>银行间市场结算速度</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>140</y>
         <width>91</width>
         <height>32</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>T+0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>T+1</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>code_text</tabstop>
  <tabstop>buy_clean_price_text</tabstop>
  <tabstop>buy_date_text</tabstop>
  <tabstop>sell_clean_price_text</tabstop>
  <tabstop>sell_date_text</tabstop>
  <tabstop>buy_date_text_2</tabstop>
  <tabstop>option</tabstop>
  <tabstop>clean</tabstop>
  <tabstop>hpy_text</tabstop>
  <tabstop>yield_if_exercised</tabstop>
  <tabstop>yield_at_buy</tabstop>
  <tabstop>hpy_annualized_text</tabstop>
  <tabstop>pushButton_clear_yield</tabstop>
  <tabstop>pushButton_yield</tabstop>
  <tabstop>pushButton_quote</tabstop>
  <tabstop>pushButton_basicinfo</tabstop>
  <tabstop>pushButton_repo_hpy</tabstop>
  <tabstop>pushButton_hpy</tabstop>
  <tabstop>pushButton_clear</tabstop>
  <tabstop>code_ib</tabstop>
  <tabstop>code_sh</tabstop>
  <tabstop>code_sz</tabstop>
  <tabstop>clean_price_ib</tabstop>
  <tabstop>clean_price_sh</tabstop>
  <tabstop>clean_price_sz</tabstop>
  <tabstop>full_price_ib</tabstop>
  <tabstop>full_price_sh</tabstop>
  <tabstop>full_price_sz</tabstop>
  <tabstop>yield_ib</tabstop>
  <tabstop>yield_sh</tabstop>
  <tabstop>yield_sz</tabstop>
  <tabstop>category</tabstop>
  <tabstop>settlement</tabstop>
  <tabstop>location</tabstop>
  <tabstop>platform</tabstop>
  <tabstop>quote_convention</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
